ocs_samples/library_samples/Python3/ocs_sample_library_preview/Dataviews.py

```diff
@@ -247,6 +247,8 @@ def getDataviewPreview(
         value_class=None,
     ):
         """
+        --- Deprecated ---
```

Comment:
Any reason we should keep a deprecated function in a Sample for a product currently in preview?

Reply:
Good question. This one is still in the currently published API ref, while the new parameters GetData do not show up yet. So I'll conservatively leave this in and remove it once both doc is updated and endpoint is removed.

Reply:
Removed